### PR TITLE
feat: add no-attr rule

### PR DESCRIPTION
**Pull request by nemmip**
#### Pull Request purpose

[ ] Documentation update
[ ] Bug fix
[X] New rule
[ ] Updates an existing rule
[ ] Add something to the core
[ ] Other, please explain:

#### Checklist

[ ] Update CHANGELOG.md
[ ] Update task status in release plan (README)
[ ] Docs added (/docs/rules/<rule_name>)
[ ] Docs ready to be added to website (ec0lint.com)

### CODE REVIEW

File: docs/rules/no-attr.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# no-attr
+
+Disallows the [`.attr`](https://api.jquery.com/attr/)/[`.removeAttr`](https://api.jquery.com/removeAttr/) methods and `$.attr`/[`$.removeAttr`](https://api.jquery.com/jQuery.removeAttr/) utilies. Prefer `Element#getAttribute`/`setAttribute`/`removeAttribute`.
+
```
Comment (malwinq) [inline docs/rules/no-attr.md:10]: What's the CO2 reduction?

File: lib/rules/index.js [hunk 2]
```diff
@@ -15,8 +15,7 @@ module.exports = new LazyLoadingRuleMap(Object.entries({
     "no-ajax": () => require("./no-ajax"),
     "no-ajax-events": () => require("./no-ajax-events"),
     "no-and-self": () => require("./no-and-self"),
-    "no-ajax": () => require("./no-ajax"),
-    "no-ajax-events": () => require("./no-ajax-events"),
+    "no-attr": () => require("./no-attr"),
     "no-moment-js": () => require("./no-moment-js"),
```
Comment (malwinq) [inline lib/rules/index.js:19]: Why two rules are deleted?

File: lib/rules/no-attr.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                    return;
+                }
+                const name = node.callee.property.name;
+                if (!methods.includes(name)) {
+                    return;
+                }
+                if (utils.isjQuery(context, node.callee)) {
+                    context.report({
+                        node: node,
+                        message: 'Prefer `Element#getAttribute`/`setAttribute`/`removeAttribute`',
```
Comment (malwinq) [inline lib/rules/no-attr.js]: We can think of changing it a bit

File: docs/rules/no-attr.md (hunk 1 above)
Comment (nemmip) [reply docs/rules/no-attr.md:10]: Docs are not done yet

File: lib/rules/index.js (hunk 2 above)
Comment (nemmip) [reply lib/rules/index.js:19]: Replied here https://github.com/ec0lint/ec0lint/pull/66

**Review by malwinq — APPROVED** [review]

File: lib/rules/no-attr.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+/** @type {import('../shared/types').Rule} */
+module.exports ={
+    meta: {
+        type: 'suggestion',
+        docs: {
+            description: 'Disallows attr utilities: attr, removeAttr. Prefer `Element#getAttribute`/`setAttribute`/`removeAttribute`.',
+            category: "Code improvements",
+            recommended: false,
+            url: "https://eslint.org/docs/rules/no-attr"
```
Comment (malwinq) [inline lib/rules/no-attr.js]: change to ec0lint

**Pull request merged**